nametoaddr.c

```diff
@@ -126,7 +126,14 @@ pcap_nametoaddr(const char *name)
 	bpf_u_int32 **p;
 	struct hostent *hp;
 
+#ifdef _WIN32
+#pragma warning (push)
+#pragma warning (disable: 4996) /* disable MSVC's gethostbyname() warning here */
+	if ((hp = gethostbyname(name)) != NULL) {
+#pragma warning (pop)
```

Comment:
I was thinking of that before. But I found the returned ``h_addr_list`` is hard to handle. I even doubt that is there a memory leak here? Because ``h_addr_list``'s memory is never freed after the function call. So i'm stuck here.